src/binary-reversal/index.js

```diff
@@ -3,6 +3,20 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function convertToReveredBinary(value) {
```

Comment:
Do you mean `reversed`? Sure do like some `revered` binary myself.

Reply:
Yes, I meant reversed. A typographic error :)